webviz_config/plugins/plugin_utils.py

```diff
@@ -0,0 +1,27 @@
+import warnings
+
+def write_metadata(distributions, metadata):

```

Comment:
This function we might want to give another, since it also loads the plugins (even though it at the same time collect metadata about them).

I also _think_ that it is the module having the `globals()` calls defined in it (i.e. this module in this case) that will get them in their namespace, and not modules importing the function and calling it. Regardless it makes it difficult to understand what is available in which namespace. Maybe the best and most explicit would be that this function also returns references to the loaded plugins, and then `plugins/__init__.py` which calls this function, is responsible for putting the plugins in its namespace.